### PR TITLE
feat(navigation-links): homogenization - INNO-760

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Homogenization of component

Note:
* I can not launch test on my environment for now, so I could not update the screenshots...

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] I have declared `@define mycomponent` in the SCSS file
- [x] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [x] I follow the naming guidelines
- [x] the component supports composition
- [x] there are no hardcoded strings (all content come from the context)
- [x] I have filled the README.md file (at least a few lines)
- [x] My component is listed in the root README
- [x] My PR has the right label(s)

### CODE REVIEW

File: framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
-<nav class="ecl-navigation-list-wrapper">
+
+{# Internal properties #}
+
+{% set _css_class = '' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_class is defined %}
```
Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig]: `extra_class` should be replaced by `extra_classes` here

File: framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print result #}
+
+<nav class="ecl-navigation-list-wrapper {{ _css_class }}" {{ _extra_attributes }}>
```
Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig]: `ecl-navigation-list-wrapper` could be integrated within `_css_class`

File: framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
   <h2 class="ecl-sr-only">{{ title|default('Navigation') }}</h2>
   <ul class="ecl-navigation-list ecl-navigation-list--{{ variant|default('default') }}">
     {% for link in links %}
-      {% set activeClass = link.isActive is defined and link.isActive == true? ' ecl-navigation-list__link--active' : '' %}
+      {% set active_class = link.is_active is defined and link.is_active == true ? ' ecl-navigation-list__link--active' : '' %}
```
Comment (yhuard) [inline framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig]: Internal properties should be prepended with an underscore

**Review by yhuard — CHANGES_REQUESTED** [review]
Some convention naming things + the reference screenshots need to be updated

File: framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig (hunk 3 above)
Comment (emeryro) [inline framework/components/ecl-navigation/ecl-navigation-lists/ecl-navigation-lists.twig]: this is not an internal property

**Comment by emeryro** [comment]
Screenshots updated

**Review by yhuard — APPROVED** [review]

**Pull request merged**